Source/Project64/UserInterface/Debugger/Debugger-Symbols.h

```diff
@@ -20,7 +20,22 @@ class CDebugSymbols :
     public CDialogResize<CDebugSymbols>
 {
 private:
+	enum {
+		m_SymbolsListView_Col_Address,
+		m_SymbolsListView_Col_Type,
+		m_SymbolsListView_Col_Name,
+		m_SymbolsListView_Col_Value,
+		m_SymbolsListView_Col_Description,
+		m_SymbolsListView_Num_Columns
+	};
```

Comment:
The `m_` prefix is not needed here. Also please use spaces instead of tabs. Everything looks good otherwise 👍